File: output/tbMateria_tbAlumno.java
public class tbMateria_tbAlumno{
	private int idMateria;
	private int idAlumno;
	private  calificacion;
	private String estado;
	private int numIntentos;

	public int getIdMateria(){
		return this.idMateria;
	}
	public void setIdMateria(int idMateria){
		this.idMateria=idMateria;
	}
	public int getIdAlumno(){
		return this.idAlumno;
	}
	public void setIdAlumno(int idAlumno){
		this.idAlumno=idAlumno;
	}
	public  getCalificacion(){
		return this.calificacion;
	}
	public void setCalificacion( calificacion){
		this.calificacion=calificacion;
	}
	public String getEstado(){
		return this.estado;
	}
	public void setEstado(String estado){
		this.estado=estado;
	}
	public int getNumIntentos(){
		return this.numIntentos;
	}
	public void setNumIntentos(int numIntentos){
		this.numIntentos=numIntentos;
	}
}

File: output/tbMateria_tbAlumno.php
<?php
class tbMateria_tbAlumno{
	private  $idMateria;
	private  $idAlumno;
	private  $calificacion;
	private  $estado;
	private  $numIntentos;

	public function getIdMateria(){
		return $this->idMateria;
	}
	public  function setIdMateria($idMateria){
		$this->idMateria=$idMateria;
	}
	public function getIdAlumno(){
		return $this->idAlumno;
	}
	public  function setIdAlumno($idAlumno){
		$this->idAlumno=$idAlumno;
	}
	public function getCalificacion(){
		return $this->calificacion;
	}
	public  function setCalificacion($calificacion){
		$this->calificacion=$calificacion;
	}
	public function getEstado(){
		return $this->estado;
	}
	public  function setEstado($estado){
		$this->estado=$estado;
	}
	public function getNumIntentos(){
		return $this->numIntentos;
	}
	public  function setNumIntentos($numIntentos){
		$this->numIntentos=$numIntentos;
	}
}
?>